hybrid_content_manager/hybridmanager.py

```diff
@@ -186,13 +190,29 @@ def root_manager(self):
     def _extra_root_dirs(self):
         return [base_directory_model(path) for path in self.managers if path]
 
+    def _validate_path(self, prefix, path):
+        validator = self.path_validator.get(prefix, lambda path: True)
+
+        path_is_valid = validator(path)
+
+        # Ensure return type is bool
+        if type(path_is_valid) is not bool:
```

Comment:
Throw an error

Reply:
Actually let's remove it. This check doesn't belong here we should find a better pattern to validate all of the configurations at once